connectorlog.go

```diff
@@ -1,9 +1,10 @@
 package insights
 
 import (
+	"bytes"
+	"encoding/json"
 	"fmt"
-	"net/http"
-	"time"
+	tim "time"
```

Comment:
Waarom moet time een alias krijgen?

Reply:
Persoonlijk vind ik het duidelijker als we de imports niet aliasen, maar gewoon de variabelen dan `tim` of `t` noemen

Reply:
> Persoonlijk vind ik het duidelijker als we de imports niet aliasen, maar gewoon de variabelen dan `tim` of `t` noemen

Mja, ik gebruik nu de regel dat top-level declarations wel voorrang krijgen op imports, maar declaraties binnen functies niet. Jij zou imports voorrang geven op alles?

Reply:
Ah ja, ik zie nu pas dat je een time struct declareert, dacht dat het voor variabele namen was. Is wel ok hoor, niet echt belangrijk